src/orquestra/sdk/_base/_in_process_runtime.py

```diff
@@ -161,11 +159,11 @@ def get_available_outputs(
             # "orquestra.sdk._base._traversal".
 
             artifact_nodes = [wf_def.artifact_nodes[id] for id in inv.output_ids]
-            packed_artifact = [
-                artifact
-                for artifact in artifact_nodes
-                if artifact.artifact_index is None
-            ][0]
+            packed_nodes = [n for n in artifact_nodes if n.artifact_index is None]
+            assert len(
+                packed_nodes
+            ), f"Task invocation should have exactly 1 packed output. {inv.id} has {len(packed_nodes)}: {packed_nodes}"  # noqa: E501
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏all non-zero values are truthy
```suggestion
            assert len(
                packed_nodes
            ) == 1, f"Task invocation should have exactly 1 packed output. {inv.id} has {len(packed_nodes)}: {packed_nodes}"  # noqa: E501
```

Reply:
Fixed in cf600b6.